src/_pytest/unittest.py

```diff
@@ -115,6 +115,8 @@ def _addexcinfo(self, rawexcinfo):
         rawexcinfo = getattr(rawexcinfo, "_rawexcinfo", rawexcinfo)
         try:
             excinfo = _pytest._code.ExceptionInfo(rawexcinfo)
+            excinfo.value
```

Comment:
Why the noops? Left from debugging? If not, please add a quick comment explaining them